docs/reference/cluster-operator/examples.md

```diff
@@ -67,14 +67,14 @@ spec:
   # address: '10.42.15.23:2379,10.42.12.22:2379,10.42.13.16:2379' # You can specify individual IPs of the etcd servers
 ```
 
-If using Etcd with mTLS, you need to specify the secret that hold the
-certificates with the following parameters:
+If using Etcd with mTLS, it is required to create the secret with the TLS
+material on the same namspace as the StorageOSCluster resource. Reference its
+name with the following parameter.
```

Comment:
```suggestion
name with the following parameter:
```

Reply:
The linter complained when I put : at the end of a paragraph.